packages/ses/demos/challenge/README.md

```diff
@@ -233,7 +232,7 @@ demonstrate the success or failure of a timing-based attack.
 The first version of this demo implemented ``delayMS()`` with a Promise that
 resolved at some point in the future (via a ``setTimeout()`` callback). In
 that version, ``guess()`` returned a Promise. Richard Gibson exploited this
-in a [clever attack](https://github.com/Agoric/ses-shim/issues/8) that submitted
+in a [clever attack](https://github.com/endojs/endo/issues/8) that submitted
```

Comment:
Should link to @gibson042 's attack at https://github.com/endojs/endo/issues/191